### ISSUE TITLE
Training example

### ISSUE THREAD

**Issue by wenbostar** [lkytal/PredFull#1]
Could you add an example to show how to train a model from scratch?

Bo

**Comment by Giels** [comment]
There are two basic ways to do that. The first is:
1. Load the model from the repository
2. Wipe out the weights
3. Train as usual

```
        pm2 = k.models.load_model("PredFull/pm.h5", compile=0)
```
[...]
```
            for layer in pm.layers:
                params = []
                if hasattr(layer, 'kernel_initializer'): 
                    w = layer.kernel_initializer(layer.weights[0].shape)
                    params.append(w)
                if hasattr(layer, 'bias_initializer'):
                    b = layer.bias_initializer(layer.weights[1].shape)
                    params.append(b)
                if len(params) > 0:
                    layer.set_weights(params)

pm.compile(optimizer=k.optimizers.Adam(lr=3e-4), loss=cos_loss, metrics=[cos_metric])
```

The second is to reproduce the model in code or config using the architecture described in the `pm.h5` file. You can get the architecture details using a combination of `tables.open_file("pm.h5", "r")` and `pm.summary()` after loading the model with keras. The former is needed to know, e.g., the dimensions of the convolution kernels, which isn't provided by `summary()`.
I'm not sure how to get some of the details such as the exact activation used (summary() only gives the name of the layer and its parameters, so you know there's an `Activation(???)` but not if `???` = 'relu' or 'sigmoid' or whatever).
The model seems to be different from the paper so this is the most reliable way.
To verify you got the arch right you can compare the output of `pm.summary()` between the pretrained model and your own.
I might post my code for the arch after I confirm the training process works the same and I finish discussing any unclear details with the authors if applicable.

**Comment by lkytal** [comment]
Close for training example already provided
